### PR TITLE
dashboard: display key configuration options vertically

### PR DESCRIPTION
**Pull request by dominic**

### CODE REVIEW

**Comment by dominic** [comment]
screen shot 2016-11-10 at 6 38 15 pm

**Comment by dominic** [comment]
PTAL

File: dashboard/src/features/shared/components/XpubField/XpubField.jsx [hunk 1]
```diff
@@ -49,40 +49,45 @@ class XpubField extends React.Component {
       this.setState({ [typeProps.value]: value })
     }
 
+    const fields = {
+      'mockhsm': <SelectField options={mockhsmKeys}
```
Comment (jeffomatic) [inline dashboard/src/features/shared/components/XpubField/XpubField.jsx]: Does it work to set autoFocus on these when they appear?

Comment (dominic) [inline dashboard/src/features/shared/components/XpubField/XpubField.jsx]: Good idea. It only autofocuses after the first change.

**Pull request merged**